cookbooks/fb_helpers/libraries/node_methods.rb

```diff
@@ -892,9 +892,9 @@ def attr_lookup(path, delim: '/', default: nil)
       # implicit-begin is a function of ruby2.5 and later, but we still
       # support 2.4, so.... until then
       node_path.inject(self) do |location, key|
-        begin
+        if key.respond_to?(:to_s) && location.respond_to?(:attribute?)
```

Comment:
This seems reasonable to me, but calling @davide125 who wrote this code to see if there was some magical reason he did `begin..rescue`.

Reply:
I don't remember making a conscious decision on this (hell, I don't remember writing this at all lol), and I suspect this was mostly copypasta from somewhere else. I think this is fine, as long as the various tests in `fb_helpers` still pass.

Reply:
> This seems reasonable to me, but calling @davide125 who wrote this code to see if there was some magical reason he did begin..rescue.


https://www.youtube.com/watch?v=XkfO8c8MlKU

> I don't remember making a conscious decision on this (hell, I don't remember writing this at all lol), and I suspect this was mostly copypasta from somewhere else. I think this is fine, as long as the various tests in fb_helpers still pass.

Awesome. Thanks for the taking a gander at this!